_posts/2018-03-10-meetup-march.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Meetup Summary (March, 2018"
+---
+
+The March Monthly meetup was held at Adlife Plaza. Members started
+streaming in from 1400Hrs.
+
+Black Panther is a recent film based on a Marvel Character by the
+same name. It is basically about a country in Africa, that was
+successfully able to avoid colonization and has been hiding its
+technological prowess from the world.
+
+The movie has been lauded by critics and audiences all over. Some
+of the book club members had also watched the film, and there were
+mixed reactions to it.
+
+The people who didn't like the film felt like it focussed a lot
+on what an American's modern Africa looked like. This resulted in
+too much similarity to American scifis than would be comfortable.
+
+Furthermore, the movie focuses a lot on African culture. This is a
```

Comment:
Oh my Jesus! Ummm... Phrase this well. [sic]You sound evil.

Put your ponts well in a manner that it hits home well. This paragraph doesn't do that :(